site/en/docs/extensions/mv3/user_interface/index.md

```diff
@@ -435,43 +461,142 @@ pages.
 </html>
 ```
 
-[1]: /docs/extensions/reference/action
-[2]: /docs/extensions/mv3/samples#search:drink
-[3]: /docs/extensions/reference/action#method-setBadgeText
-[4]: /docs/extensions/reference/action#method-setBadgeBackgroundColor
-[5]: /docs/extensions/reference/action
-[6]: /docs/extensions/reference/declarativeContent
-[7]: /docs/extensions/reference/runtime#event-onInstalled
-[8]: /docs/extensions/mv3/background_pages
-[10]: /docs/extensions/reference/action#method-show
-[11]: /docs/extensions/reference/action#method-show
-[12]: /docs/extensions/reference/action#method-hide
-[13]: /docs/extensions/mv3/samples#search:mappy
-[15]: /docs/extensions/reference/action
-[16]: /docs/extensions/reference/action
-[17]: /docs/extensions/mv3/samples#search:drink
-[18]: /docs/extensions/reference/action#method-setPopup
-[20]: /docs/extensions/reference/action
-[21]: /docs/extensions/reference/action
-[22]: /docs/extensions/reference/action#method-setTitle
-[23]: /docs/extensions/reference/action#method-setTitle
-[24]: /docs/extensions/reference/i18n
-[25]: /docs/extensions/mv3/i18n-messages
-[26]: /docs/extensions/reference/omnibox
-[27]: https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/api/omnibox/new-tab-search
-[28]: /docs/extensions/reference/omnibox#event-onInputEntered
-[29]: /docs/extensions/reference/contextMenus
-[30]: /docs/extensions/reference/contextMenus#method-create
-[31]: /docs/extensions/mv3/migrating_to_service_workers
-[32]: /docs/extensions/reference/runtime#event-onInstalled
-[33]: /docs/extensions/examples/api/contextMenus/global_context_search/locales.js
-[34]: /docs/extensions/reference/commands
-[35]: /docs/extensions/mv3/samples#search:tab%20flipper
-[36]: /docs/extensions/reference/commands#event-onCommand
-[37]: /docs/extensions/mv3/migrating_to_service_workers
-[38]: /docs/extensions
-[39]: /docs/extensions/mv3/user_interface#browser
-[40]: /docs/extensions/mv3/migrating_to_service_workers
-[42]: /docs/extensions/mv3/override
-[43]: /docs/extensions/mv3/user_interface/#popup
+### Notifications
+
+You can communicate relevant information to users by displaying notifications directly in their
+system tray.
+
+To use the [`notifications`][api-notif] API, you must declare the `"notifications"` permission in
+the manifest.
+
+```json/5
+{ //manifest.json
+  "name": "Drink Water Event Popup",
+...
+  "permissions": [
+    "alarms",
+    "notifications",
+    "storage"
+  ],
+ ...
+}
+```
+
+In the Drink Water extension, you can remind users when to hydrate.
```

Comment:
This line is generally restatting the purpose of the extension. Since this seems like it's a step in a mini tutorial, I'd suggest changing this prompt to something more contextually relevant.

```suggestion
Once the permission is declared, you can display a notification by calling [`notifications.create()`][notifications-create].

[notifications-create]: /docs/extensions/reference/notifications#method-create
```